services/openshiftbuilddeploy/src/index.ts

```diff
@@ -333,7 +333,8 @@ const messageConsumer = async msg => {
             "name":openshiftProject,
             "labels": {
               "lagoon.sh/project": safeProjectName,
-              "lagoon.sh/environment": safeBranchName
+              "lagoon.sh/environment": safeBranchName,
+              "lagoon.sh/environmenttype": lagoonEnvironmentType
```

Comment:
```suggestion
              "lagoon.sh/environmentType": lagoonEnvironmentType
```
we use already this way in https://github.com/amazeeio/lagoon/blob/6d08071fad9619d9fe2bd54a77f18124a66c9691/images/kubectl-build-deploy-dind/helmcharts/cli/templates/_helpers.tpl#L58

Reply:
:+1: